webextension/selector/shooter.js

```diff
@@ -78,7 +78,9 @@ window.shooter = (function () { // eslint-disable-line no-unused-vars
       selectedPos,
       shotId: shot.id,
       shot: shot.asJson()
-    }));
+    })).then((url) => {
```

Comment:
Yes, it does.

Also, looking more closely at the code I realize there's a problem here – `catcher.watchPromise()` should wrap the entire promise, including the new .then expression.